samples/ipc/ipc_service/README.rst

```diff
@@ -66,10 +66,16 @@ Building and running
 **nRF5340 DK**
 
 You can build the sample using either the RPMsg or ICMSG backends, as follows:
+The RPMsg backend is the default one, you can use the following command:
```

Comment:
Just suggesting a bit different wording. Still @peknis has a final word on this.
```suggestion
For the RPMsg backend, that is the default one, you can use the following command:
```

Reply:
Now, you have two sentences with colon at the end, which is not good. Suggesting the following:

You can build the sample using either the RPMsg or ICMSG backend.
For the default RPMsg backend, use the following command: